mmf/utils/wandb_logger.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
We should move this to `mmf/utils/logger.py` to keep things around logger together.

Reply:
Fixed.